scripts/partition_manager.py

```diff
@@ -0,0 +1,406 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+
+import argparse
+import yaml
+import re
+from os import path
+
+
+def remove_item_not_in_list(list_to_remove_from, list_to_check):
```

Comment:
items, not item.

And it does more than this, since it is 'app' aware, so this abstraction isn't really working. This function should be inlined I suspect.